### PR TITLE
Document terraform scanning

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

File: docs/configure/ci/configuration.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ meta:
   - name: description
     content: "Fairwinds Insights | Documentation: How to configure the CI integration"
 ---
-# Insights Continuous Integration (CI)
+# fairwinds-insights.yaml
 The Insights CI integration relies on the `fairwinds-insights.yaml` configuration file in the root of your infrastructure-as-code repository to understand exactly what needs to be scan.
```
Comment (lizcoolman) [inline docs/configure/ci/configuration.md:7]: update "scan" to "scanned"

File: docs/configure/ci/configuration.md [hunk 2]
```diff
@@ -105,6 +105,15 @@ manifests:
       repo: https://helm.nginx.com/stable
 ```
 
+## Scanning Terraform Files
+You can scan Terraform files by specificing a list of directories to scan:
```
Comment (lizcoolman) [inline docs/configure/ci/configuration.md:109]: update to "Scan Terraform files by specifying a list of directories to scan:"

**Review by lizcoolman — COMMENTED** [review]
Added comments for review

File: docs/configure/ci/configuration.md (hunk 2 above)
Comment (rbren) [reply docs/configure/ci/configuration.md:109]:
```suggestion
Scan Terraform files by specificing a list of directories to scan:
```

**Comment by rbren** [comment]
Thanks! Good eye

File: docs/configure/ci/configuration.md (hunk 2 above)
Comment (lizcoolman) [reply docs/configure/ci/configuration.md]: @rbren update "specificing" to "specifying"

File: docs/configure/ci/configuration.md [hunk 3]
```diff
@@ -105,6 +105,15 @@ manifests:
       repo: https://helm.nginx.com/stable
 ```
 
+## Scanning Terraform Files
+Scan Terraform files by specificing a list of directories to scan:
```
Comment (rbren) [inline docs/configure/ci/configuration.md:109]:
```suggestion
Scan Terraform files by specifying a list of directories to scan:
```

**Pull request closed without merging**